README.md

```diff
@@ -1,11 +1,17 @@
 ![Open Data Service (ODS)](https://github.com/jvalue/open-data-service/workflows/Open%20Data%20Service%20(ODS)/badge.svg?branch=master)
+[![on gitter](https://badges.gitter.im/jvalue-ods/community.svg)](https://gitter.im/jvalue-ods/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
 # Open Data Service (ODS)
 
 The Open Data Service (ODS) is an application which can collect data from multiple sources simulataneously, process that data and then offer an improved (or "cleaned") version to its clients.
 *We aim to establish the ODS as **the** go-to place for using Open Data!*
 
+## Contact us
+
+If you have any questions or would like to contact us, you can easily reach us via [gitter channel](https://gitter.im/jvalue-ods/community). Issues can be reported via [GitHub](https://github.com/jvalue/open-data-service/issues).
```

Comment:
Maybe we should add a public email address here as well?

Reply:
Yeah, we first need one I guess. Postponed ;)